src/components/ChartBar/ChartBar.js

```diff
@@ -29,6 +29,13 @@ function ChartBar(props) {
     }
   };
 
+  // Converts number to simpler decimal
+  function simpleNum(){
+    var num = state.updatedValue * .00000001
```

Comment:
I'm seeing $55 bil instead of 5.5 so I think this needs one more 0 (.000000001). Otherwise looks good, so much cleaner.

Reply:
Okay, I have to work on this. When I do .000000001 I get 0.6 Million when things are millions.

Reply:
ahh ok. Maybe there's a way to combine those two functions then, adding a check to see if it should be Million or Billion.